nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="node-maintenance-operator"]
+= Using the Node Maintenance Operator to place nodes in maintenance mode
+include::modules/common-attributes.adoc[]
+include::modules/virt-document-attributes.adoc[]
+:context: node-maintenance-operator
+toc::[]
+
+You can use the Node Maintenance Operator to place nodes in maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of {VirtProductName} installation.
+
+[NOTE]
+====
+If you have installed {VirtProductName}, you must use the Node Maintenance Operator that is bundled with it. 
+====
+
+include::modules/eco-about-node-maintenance-standalone.adoc[leveloffset=+1]
+
+include::modules/eco-maintaining-bare-metal-nodes.adoc[leveloffset=+1]
+
+You can install the Node Maintenance Operator using the web console or the OpenShift CLI (`oc`).
+
+include::modules/eco-node-maintenance-operator-installation-web-console.adoc[leveloffset=+1]
+
+include::modules/eco-node-maintenance-operator-installation-cli.adoc[leveloffset=+1]
+
+Place a node into maintenance from the CLI.
```

Comment:
This seems a bit out of place in the rendered output - it almost looks like a next step for the preceding procedure, but is not formatted as such, and is a bit easy to lose. Consider maybe expanding  it into more of an intro ("You can ....."), or maybe removing it entirely?

Same for lines 30, 34, and 40.